acceptance/tests/ticket_1429_facter_with_strict_flag.rb

```diff
@@ -1,11 +0,0 @@
-test_name 'ticket FACT-1476 - C98098 facter cli with --strict flag'
```

Comment:
Looks like this test became "acceptance/tests/options/strict.rb". Would you please update the commit message in git to that effect -- it isn't immediately clear from the diff that this is essentially a rename.